src/box/box.cc

```diff
@@ -3690,6 +3690,17 @@ tx_prio_cb(struct ev_loop *loop, ev_watcher *watcher, int events)
 	(void) loop;
 	(void) events;
 	struct cbus_endpoint *endpoint = (struct cbus_endpoint *)watcher->data;
+#ifndef NDEBUG
+	/*
+	 * The sleep is legal because it is not a fiber sleep. It puts the
+	 * entire thread to sleep to simulate it being slow. It can happen in
+	 * reality if the thread somewhy isn't scheduled for too long.
+	 */
+	struct errinj *inj = errinj(ERRINJ_TX_DELAY_PRIO_ENDPOINT,
+				    ERRINJ_DOUBLE);
+	if (inj->dparam != 0)
+		usleep(inj->dparam * 1000000);
```

Comment:
I haven't read the code below yet, but why didn't ERROR_INJECT_SLEEP and a errinj with a bool param fit?

Reply:
`ERROR_INJECT_SLEEP` has too small timeout - 1ms. To make the bug appear almost always I wanted to use 100ms. With that delay it fails on my machine always when run by test-run. On the other hand, a non-boolean value gives you a flexibility to choose another timeout. For instance, in a console you might want to use order of seconds to catch it manually. For some other bug you might need order of microseconds. TLDR: I find it more flexible.